pkg/preparer/setup.go

```diff
@@ -161,7 +161,7 @@ func getTLSClient(certFile, keyFile, caFile string) (*http.Client, error) {
 	}
 
 	if len(certs) == 0 && cas == nil {
-		return http.DefaultClient, nil
+		return &tls.Config{}, nil
```

Comment:
`nil` is a perfectly fine value to return to get a [default config](https://golang.org/pkg/net/http/#Transport).